misc/agent-introspection-validator/agent-introspection-validator.go

```diff
@@ -252,6 +242,23 @@ func metadataResponseOnce(client *http.Client, endpoint string, respType string)
 		return nil, fmt.Errorf("%s: unable to read response body: %v", respType, err)
 	}
 
+	fmt.Printf("Received %s metadata response: %s \n", respType, body)
+
+	// verify metadata response
+	if respType == tasksMetadataRespType {
+		err = verifyTasksMetadata(body)
+		if err != nil {
```

Comment:
Why we remove sleep to wait for volume code blob and add this part? Are they related?